lib/PHPCfg/Parser.php

```diff
@@ -1028,6 +1029,11 @@ protected function parseExpr_Exit(Expr\Exit_ $expr)
             $e = $this->readVariable($this->parseExprNode($expr->expr));
         }
 
+        $this->block->children[] = new Op\Terminal\Exit_($expr->expr, []);
+        // Dump everything after the exit
+        $this->block = new Block();
+        $this->block->dead = true;
+
         return new Op\Expr\Exit_($e, $this->mapAttributes($expr));
```

Comment:
Now that `Exit_` exists as a terminal, the `Expr\Exit_` class should be dropped.